zzz_generated/app-common/app-common/zap-generated/af-structs.h

```diff
@@ -450,7 +450,6 @@ typedef struct _ProviderLocation
 typedef struct _ReadAttributeStatusRecord
 {
     chip::AttributeId attributeId;
-    chip::Protocols::InteractionModel::Status status;
```

Comment:
Why are these changes happening?

Reply:
So I see that Status is actually not defined in ` src/app/zap-templates/zcl/data-model/silabs/types.xml` which is an `ENUM8` so my guess is it should have been `uint8_t status;` . Not sure why it said `chip::Protocols::InteractionModel::Status status;` before which seems off.

Reply:
My point is: why is this being changed even though the template has not changed?  What was causing this to appear this way before but not now?

Reply:
As to how it appeared that way before: It just took type as `status` and `override.js#atomicType` returned that using `case 'status':
    return 'chip::Protocols::InteractionModel::Status';`

Reply:
Should preserve that behavior somehow?  Though maybe it does not matter if no one is using these generated structs for anything....

In any case, thank you for hunting down where that came from.  Should that line in override.js be removed now?